standalone_app/src/components/PlotImportance.tsx

```diff
@@ -59,17 +59,15 @@ export const PlotImportance: FC<{ study: Study }> = ({ study }) => {
   }, [nObjectives, importance, theme.palette.mode])
 
   return (
-    <Card>
-      <CardContent>
-        <Typography
-          variant="h6"
-          sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
-        >
-          Hyperparameter Importance
-        </Typography>
-        <Box id={plotDomId} sx={{ height: "450px" }} />
-      </CardContent>
-    </Card>
+    <Box>
+      <Typography
+        variant="h6"
+        sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
+      >
+        Hyperparameter Importance
+      </Typography>
+      <Box id={plotDomId} sx={{ height: "450px" }} />
+    </Box>
```

Comment:
How about using `<Fragment>` instead?
https://react.dev/reference/react/Fragment

```suggestion
    <>
      <Typography
        variant="h6"
        sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
      >
        Hyperparameter Importance
      </Typography>
      <Box id={plotDomId} sx={{ height: "450px" }} />
    </>
```